data/editor.config

```diff
@@ -558,281 +543,212 @@
-        <include field="internet" />
+        <include group="poi_internet" />
       </type>
       <type id="shop-mobile_phone" group="shop">
-        <include group="poi" />
-        <include field="internet" />
+        <include group="poi_internet" />
       </type>
       <type id="shop-nutrition_supplements" group="shop">
-        <include group="poi" />
-        <include field="internet" />
+        <include group="poi_internet" />
       </type>
       <type id="shop-optician" group="shop">
-        <include group="poi" />
-        <include field="internet" />
+        <include group="poi_internet" />
       </type>
       <type id="shop-outdoor" group="shop">
-        <include group="poi" />
-        <include field="internet" />
+        <include group="poi_internet" />
       </type>
       <type id="shop-perfumery" group="shop">
-        <include group="poi" />
-        <include field="internet" />
+        <include group="poi_internet" />
       </type>
       <type id="shop-second_hand" group="shop">
-        <include group="poi" />
-        <include field="internet" />
+        <include group="poi_internet" />
       </type>
       <type id="shop-sewing" group="shop">
-        <include group="poi" />
-        <include field="internet" />
+        <include group="poi_internet" />
       </type>
       <type id="shop-shoes" group="shop">
-        <include group="poi" />
-        <include field="internet" />
+        <include group="poi_internet" />
       </type>
       <type id="shop-sports" group="shop">
-        <include group="poi" />
-        <include field="internet" />
+        <include group="poi_internet" />
       </type>
       <type id="shop-supermarket" group="big_shop">
-        <include group="poi" />
-        <include field="internet" />
+        <include group="poi_internet" />
       </type>
       <type id="shop-tobacco" group="shop">
-        <include group="poi" />
-        <include field="internet" />
+        <include group="poi_internet" />
       </type>
       <type id="shop-toys" group="shop">
-        <include group="poi" />
-        <include field="internet" />
+        <include group="poi_internet" />
       </type>
       <type id="shop-photo" group="shop">
-        <include group="poi" />
-        <include field="internet" />
+        <include group="poi_internet" />
       </type>
       <type id="shop-travel_agency" group="shop">
-        <include group="poi" />
+        <include group="poi_internet" />
       </type>
       <type id="shop-tyres" group="shop">
-        <include group="poi" />
+        <include group="poi_internet" />
       </type>
       <type id="shop-watches" group="shop">
-        <include group="poi" />
-        <include field="internet" />
+        <include group="poi_internet" />
       </type>
       <type id="tourism-alpine_hut" group="accomodation">
-        <include group="poi" />
+        <include group="poi_internet" />
         <include field="ele" />
```

Comment:
What is the difference between "ele" vs "height" editable fields?

Reply:
ele field is elevation in meters of POI (used for a mountains), height (in meters) field is used more for height of buildings